repo/repo.go

```diff
@@ -217,17 +248,27 @@ func parseRepositoryDirectives(directives []rule.Directive) (repos []*rule.Rule,
 	return repos, nil
 }
 
+type RepoMacroDirective struct {
+	File     string
```

Comment:
optional nit: `Path` or `FilePath` might be more clear to not confuse with `rule.File` structs

Reply:
https://github.com/bazelbuild/bazel-gazelle/blob/master/rule/rule.go#L57-L61 uses `Path` and `DefName`, which would probably be good naming conventions to mirror and stay consistent with.